src/Microsoft.DotNet.Tools.Pack/Properties/launchSettings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "profiles": {
+    "dotnet-pack": {
+      "executablePath": "..\\..\\artifacts\\win7-x64\\stage2\\bin\\dotnet-pack.exe"
```

Comment:
why do we need this? `Microsoft.DotNet.Cli.Utils.Command.Create("dotnet-pack");`